src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/Totrinnskontroll/Totrinnskontrollskjema.tsx

```diff
@@ -61,87 +69,88 @@ const Totrinnskontrollskjema: React.FunctionComponent<IProps> = ({
-                                <Detail size={'small'}>Vedtaket er sendt til godkjenning</Detail>
-                            </div>
-                        </SendtTilBeslutterContainer>
-                    ) : (
-                        <>
+            error={hentFrontendFeilmelding(innsendtVedtak)}
+            legend={
+                egetVedtak ? (
+                    <SendtTilBeslutterContainer>
+                        <div>
+                            <Heading size={'medium'} level={'2'}>
+                                Totrinnskontroll
+                            </Heading>
+                            <br />
                             <BodyShort>
-                                Kontrollér opplysninger og faglige vurderinger som er gjort
+                                {formaterIsoDato(
+                                    opprettetTidspunkt,
+                                    datoformat.DATO_FORLENGET_MED_TID,
+                                    'UKJENT OPPRETTELSESTIDSPUNKT'
+                                )}
                             </BodyShort>
-
+                            <BodyShort>{saksbehandler}</BodyShort>
                             <br />
-                            <KontrollerteTrinnOverskrift>
-                                Kontrollerte trinn
-                            </KontrollerteTrinnOverskrift>
-
-                            {Object.entries(trinnPåBehandling).map(([_, trinn], index) => {
-                                return (
-                                    <TrinnStatus
-                                        kontrollertStatus={trinn.kontrollert}
-                                        navn={`${index + 1}. ${trinn.navn}`}
-                                    />
-                                );
-                            })}
-                        </>
-                    )
-                }
-            >
+                            <Detail size={'small'}>Vedtaket er sendt til godkjenning</Detail>
+                        </div>
+                    </SendtTilBeslutterContainer>
+                ) : (
+                    <>
+                        <BodyShort>
+                            Kontrollér opplysninger og faglige vurderinger som er gjort
+                        </BodyShort>
+
+                        <br />
+                        <KontrollerteTrinnOverskrift>
+                            Kontrollerte trinn
+                        </KontrollerteTrinnOverskrift>
+
+                        {Object.entries(trinnPåBehandling).map(([_, trinn], index) => {
+                            return (
+                                <TrinnStatus
+                                    kontrollertStatus={trinn.kontrollert}
+                                    navn={`${index + 1}. ${trinn.navn}`}
+                                />
+                            );
+                        })}
+                    </>
+                )
+            }
+        >
```

Comment:
Flytter denne teksten opp til legend på fieldset i stedet for å være description på radioGroup. Ble mer riktig både visuelt og mentalt

Reply:
Rent semantisk blir det ikke helt riktig å ha en hel seksjon med innhold i legend - alt det puttes inni en `<label>` 🤔

I tillegg kommer RadioGroup med sitt eget Fieldset (`legend`-prop på RadioGroup sendes direkte dit). Hva med å legge dette inni RadioGroup, før Radio-ene?

Reply:
Er enig i det! Prøvde meg på en endring nå. Hva syns du om det?

Her er skjermbilder med ny løsning:
Før
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/2028a988-e679-40ab-8eda-20d4aa800758)

Etter
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/750f7fca-2069-4641-a900-617d404c3745)


### Eget vedtak
Før
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/d5e65ebf-f872-4f65-8cd0-6cfc970fbf8b)

Etter
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/f6d0aff1-1418-473f-848e-fcb0ef294a34)